src/sidebar/components/Thread.js

```diff
@@ -113,17 +114,27 @@ function Thread({ thread, threadsService }) {
     () =>
       thread.visible && (
         <>
-          {thread.annotation && (
-            <ModerationBanner annotation={thread.annotation} />
+          {thread.annotation ? (
+            <>
+              <ModerationBanner annotation={thread.annotation} />
+              <Annotation
+                annotation={thread.annotation}
+                hasAppliedFilter={hasAppliedFilter}
+                isReply={isReply}
+                onToggleReplies={onToggleReplies}
+                replyCount={thread.replyCount}
+                threadIsCollapsed={thread.collapsed}
+              />
+            </>
+          ) : (
+            <EmptyAnnotation
+              hasAppliedFilter={hasAppliedFilter}
+              isReply={isReply}
+              onToggleReplies={onToggleReplies}
+              replyCount={thread.replyCount}
+              threadIsCollapsed={thread.collapsed}
+            />
```

Comment:
If the non-DRY-ness of these props bugs you I could extract them into a shared props object (excepting `annotation` which only goes to `Annotation`).

Reply:
I think this is fine for the moment. TS will catch any mandatory props that are missing, eg. if you add a new prop to `Annotation` that needs to be handled in both.